scheduler/client/cli/cli.go

```diff
@@ -5,23 +5,16 @@ import (
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	commoncli "github.com/twitter/scoot/common/client"
 	"github.com/twitter/scoot/common/dialer"
+	smoketest "github.com/twitter/scoot/integration-tests/smoketest"
 	"github.com/twitter/scoot/scheduler/api/thrift/gen-go/scoot"
 	"github.com/twitter/scoot/scheduler/client"
 )
 
 // Scoot API Client interface that includes CLI handling
-type CLIClient interface {
-	Exec() error
-}
-
-// Implements CLIClient - basic
-type simpleCLIClient struct {
-	rootCmd     *cobra.Command
-	addr        string
-	dial        dialer.Dialer
-	logLevel    string
-	scootClient *client.CloudScootClient
+type SimpleCLIClient struct {
```

Comment:
Could we name this schedCLI or schedClient or something more descriptive

Reply:
Makes sense, renaming it to SchedCLIClient